### ISSUE TITLE
Database is locked

### ISSUE THREAD

**Issue by Newer123** [KN4CK3R/KeePassBrowserImporter#12]
Hello,

Anytime that I am trying to import my bookmark passwords from Google Chrome I am getting an error saying "Database is locked". I have tried in my current database and another one that I've created for testing purposes and I always get the same error. Could you please assist?
Thanks,
Peter

**Comment by KN4CK3R** [comment]
Maybe because the database is locked while the plugin tries to create the entries? The plugin doesn't close the database so this must be initiated from outside. (KeePass gets locked after some time, KeePass gets locked if it hasn't the focus anymore, ... don't know what other settings could cause a database lock).

**Comment by mrsof** [comment]
Close Chrome and KeePass and retry.

**Comment by KN4CK3R** [comment]
Right, the error could be caused by Chrome and not KeePass. Just close the browser so no files are locked.

**Comment by martin-rhodes** [comment]
I also had this problem. Ending all chrome.exe processes in the Details view of Task Manager before restarting KeePass and trying again fixed it for me (some extensions like the Google Hangouts plugin will keep chrome.exe open in the background even if all Chrome browser windows are closed.)

**Comment by damienallen** [comment]
As Martin has pointed out, Chrome can run in the background and cause this error. I found that the easiet solution was to close Chrome using the system tray icon menu. From there it is possible to exit or prevent Chrome from running in the background entirely.
